src/EditorFeatures/TestUtilities/ExperimentationService/TestExperimentationServiceInternal.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.VisualStudio.Text.Utilities;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.ExperimentationService
+{
+    [Export(typeof(IExperimentationServiceInternal)), Shared]
```

Comment:
❕ Missing `[PartNotDiscoverable]`

❕ Missing importing constructor

Reply:
[PartNotDiscoverable] is not applicable here. We use it for exporting command handlers and it lose the async completion one if [PartNotDiscoverable]


❕ Missing importing constructor - fixed

---
In reply to: [208002820](https://github.com/dotnet/roslyn/pull/29016#discussion_r208002820) [](ancestors = 208002820)